mw-webapp/src/component/label/Label.module.scss

```diff
@@ -16,3 +16,8 @@ $labelMaxWidth: 130px;
   font-size: $fontSizeSmall;
   white-space: nowrap;
 }
+
+.editableText span p {
```

Comment:
Is it possible to avoid using tag in styles? Let's use classnames

Reply:
it's not possible because tag p is markdown. I don't  know how  to add classname props to markdown 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/bce811df-edd8-4aea-a337-d6db8fb995a3)

Reply:
we need to add props.classname to Label => EditableText =>  Text => renderMarkdown(props.text) and finally add to MarkdownLib? now this styles is workaround and I think there is no need to pass this classname as a prop through 3 components

Reply:
Понимаю, что не касается этого ПР, но можно вкладывать p внутрь span?

Reply:
I think its not correct. It will be better use block elemebt like div. if you want I can change to div tag

Reply:
Давай проверим не поломается ли в других местах чтото, если изменить тэг. Если что - то создай новое issue

Reply:
I have allready changed 
![image](https://github.com/user-attachments/assets/a86fc853-f331-4390-950b-986b84413a8a)